dataset_generator/fuzzing-script.py

```diff
@@ -0,0 +1,101 @@
+import random
+import os
+from typing import Union
+from nelox.Expr import Variable, Literal, List
+from nelox.nelox_token import Token
+from nelox.token_type import TokenType
+from dataset_generator.EnvStack import EnvStack
+from nelox.pretty_printer import pretty,pretty_program
+
+OP = ["+", "-", "*", "/"]
+NUM = [random.randint(1, 100) for _ in range(50)]
+var_pool = ["x", "y", "z", "a", "b", "c", "d", "e", "f"]
```

Comment:
nit: inconsistent name with `OP` and `NUM`, please follow PEP8

Reply:
Please see https://peps.python.org/pep-0008/#constants
By the way, to enforce that these are constants, you may want to make them tuples.